lib/matplotlib/texmanager.py

```diff
@@ -92,19 +112,24 @@ def get_font_config(self):
         fontconfig = [self._font_family]
         fonts = {}
         for font_family in self._font_families:
-            for font in rcParams['font.' + font_family]:
-                if font.lower() in self._font_info:
-                    fonts[font_family] = self._font_info[font.lower()]
-                    _log.debug(
-                        'family: %s, font: %s, info: %s',
-                        font_family, font, self._font_info[font.lower()])
-                    break
-                else:
-                    _log.debug('%s font is not compatible with usetex.', font)
+            if _aux_check:
+                fonts[font_family] = self._font_info[ff_val]
```

Comment:
I think you still need to check the other families, i.e. if someone has
```
rcParams["font.family"] = "palatino"
rcParams["font.monospace"] = "courier"
```
then you need to import both mathpazo (so that with rmfamily the text defaults to palatino) *and* courier (so that if the user has a `\texttt` halfway in the string they get courier).